test/index.js

```diff
@@ -1193,14 +1237,15 @@ describe('def', function() {
 
     throws(function() { $.RecordType({x: /XXX/, y: /XXX/, z: $.Any}); },
            TypeError,
-           'Invalid values\n' +
+           'Invalid value\n' +
            '\n' +
-           'The argument to ‘RecordType’ must be an object mapping field name to type.\n' +
+           'RecordType :: StrMap Type -> Type\n' +
+           '                     ^^^^\n' +
+           '                      1\n' +
            '\n' +
-           'The following mappings are invalid:\n' +
+           '1)  /XXX/ :: RegExp\n' +
```

Comment:
Currently `RecordType` has a 15-line block of custom type checking, which allows it to provide customized error messages. The new error messages are slightly less informative, but they are more consistent with the error messages produced in other contexts. On balance I think replacing the custom type checking with

```javascript
def('RecordType', {}, [StrMap(Type), Type], RecordType)
```

is worthwhile. What do you think?